lib/Url/UrlManager.php

```diff
@@ -253,6 +254,35 @@ public static function getByProfileId($profileId)
         return $instances;
     }
 
+    public function publishedInSitemap()
+    {
+        if (!$this->inSitemap()) {
+            return false;
+        }
+
+        $clangId = $this->values['clang_id'];
+        $clang   = \rex_clang::get($clangId);
+
+        if (!$clang->isOnline()) {
+            return false;
+        }
+
+        $article = \rex_article::get($this->getArticleId(), $clangId);
+        if (!$article->isOnline()) {
```

Comment:
```suggestion
        if (!$article || !$article->isOnline()) {
```

Sichergehen, dass kein Whoops passiert.

Reply:
find ich gut